### DISCUSSION TITLE
Documentation needs an Update

### DISCUSSION THREAD

**Discussion by bonsaibauer** [lexiconhq/lexicon#27] [Ideas]
Hello there, this is Philipp. I would like to express my appreciation for the great project here 🙌.

Regarding the documentation at docs.lexicon.is, I have noticed that there is a lack of information. However, I have also seen a wonderful documentation with a lot of engagement, and I am grateful for that.

I am wondering if it would be possible to create a Wiki where new changes and explanations can be updated?

- I have an idea for a post titled "How to Set Up SSL for ProseQL."
- Additionally, an article could be added on setting up Discourse with SSL using Apache configurations for Docker installation.
- I was initially confused about the difference between a tutorial and documentation.

**Comment by Kinostrome** [comment]
There will be some documentation updates that come with our version 2 release, especially related to the new EAS integration with Expo.

> I have an idea for a post titled "How to Set Up SSL for ProseQL."

Please do submit it as a PR, it would be great to check it out.

> Additionally, an article could be added on setting up Discourse with SSL using Apache configurations for Docker installation.

Personally, I don't use Apache much these days. I tend to use Caddy and nginx.

I don't know that I'd want to get into the habit of the core team needing to maintain a guide for multiple server + SSL configuration setups, as that could get quite messy (and quickly out of date).

Additionally, with the advent of LetsEncrypt and provider-based SSL, there are a lot of different ways to configure HTTPS. However, for Apache in particular, it's a great opportunity for someone from the community (such as yourself) to write an article on how to set it up.

While we have purposely stayed out of recommending a specific deployment setup for the time being, I do agree that having at least one guide could be useful. Internally, we use a combination of Cloudflare, Docker, and Caddy. However, this is a fairly opinionated setup that may even cost users money, so we probably wouldn't recommend that generally.

> I was initially confused about the difference between a tutorial and documentation.

It is fairly standard for a project to have more detailed documentation, and a separate tutorial to hand-hold users through a common scenario to familiarize them with the project setup step-by-step.
If you have any other projects that you think are a good example of structuring this more clearly, feel free to share them.

And, since you may have ideas on what would make it clearer, we would also welcome a well-written PR to change this part of the documentation as well. Everything is located in `documentation/`.

**Comment by Kinostrome** [comment]
@bonsaibauer by the way, thanks for making use of the discussions tab. You made me realize that some of the Github issues are better as discussions anyway and will be easier to find for anyone troubleshooting in the future.
